File: PharmaDicBackEnd/PharmaDicBackEnd.ApiService/Models/MedicineIngredient.cs
using System;
using System.Collections.Generic;

namespace PharmaDicBackEnd.ApiService.Models;

public partial class MedicineIngredient
{
    public int MedicineId { get; set; }

    public int IngredientId { get; set; }

    public string? Amount { get; set; }

    public virtual Ingredient Ingredient { get; set; } = null!;

    public virtual Medicine Medicine { get; set; } = null!;
}

File: PharmaDicMobile/app/src/main/java/com/example/pharmadicmobile/models/MedicineIngredient.java
package com.example.pharmadicmobile.models;

import java.io.Serializable;

public class MedicineIngredient implements Serializable {
    private int medicineId;
    private int ingredientId;
    private String amount;
    private Ingredient ingredient;
    private Medicine medicine;

    public MedicineIngredient() {}

    public int getMedicineId() { return medicineId; }
    public void setMedicineId(int medicineId) { this.medicineId = medicineId; }

    public int getIngredientId() { return ingredientId; }
    public void setIngredientId(int ingredientId) { this.ingredientId = ingredientId; }

    public String getAmount() { return amount; }
    public void setAmount(String amount) { this.amount = amount; }

    public Ingredient getIngredient() { return ingredient; }
    public void setIngredient(Ingredient ingredient) { this.ingredient = ingredient; }

    public Medicine getMedicine() { return medicine; }
    public void setMedicine(Medicine medicine) { this.medicine = medicine; }
}
